test-code.rb

```diff
@@ -0,0 +1 @@
+defined? Foo.foo()
```

Comment:
You probably didn't want to commit this right?

Reply:
oops! thanks!